tests/e2e_logging.py

```diff
@@ -434,6 +434,28 @@ def test_historical_query(network, args):
     return network
 
 
+@reqs.description("Read historical receipts")
+@reqs.supports_methods("log/private", "log/private/historical_receipt")
+def test_historical_receipts(network, args):
+    if args.consensus == "bft":
+        LOG.warning("Skipping historical queries in BFT")
+        return network
+
+    if args.package == "liblogging":
+        node = network.find_node_by_role()
+        network.txs.issue(network, number_txs=1)
+        first_msg = network.txs.priv[1][0]
+        first_receipt = network.txs.get_receipt(
```

Comment:
Can we add a call to `/receipt/verify` to confirm this is a valid receipt?

Reply:
Won't work all that well until that supports historical queries as well, unless it's still in cache. I'd rather do the offline validation in the test code first, which I think is much more useful and is a better test as well.

Reply:
Done